src/pages/Profile.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div>
     <h1>Profile Page</h1>
+    <router-link :to="'/profile/' + $auth.currentUser.uid ">My Profile</router-link>
```

Comment:
lets use the fancy template literals!

```js
`/profile/${$auth.currentUser.uid}`
```

Reply:
whether we use the state for the comparing UID vs auth shouldn't matter, right?

Reply:
Aggreed I would prefer to use template literals, but my highlighting wasn't working.  Do I need to put the whole thing in {} or just drop the template literals directly into the :to

Reply:
The store uid should be the same as the $auth.currentUser.uid so I could change that to the store getter for the uid.

Reply:
i don't really care where we get the uid from, mostly just wanna make sure whatever we decide is consistent as much as possible

Reply:
so i use vscode and the highlighting is awesome for vue once you install vetur plugin.  i had a hard time getting the HTML portion to syntax highlight properly in atom.  maybe we should make a vue plugin for atom to fix that lol.